Assets/General/Scripts/Health/GeneralVfxFunctionality.cs

```diff
@@ -6,6 +6,8 @@
 // Description:     Handles any general functionality related to an object with particle systems
 ******************************************************************************/
```

Comment:
I see this file is located under Assets/General/Scripts/Health. Not sure why it's under health. Maybe I did that I have no idea, I know it wasn't your fault here.